plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2036,63 @@ def coupling_parameter(
         )
 
     return coulomb_energy / kinetic_energy
+
+
+class RelaxationRates:
+    @validate_quantities(
+        n_b={"can_be_negative": False},
+        T_b={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    )
+    def __init__(
+        self,
+        species: (particles.Particle, particles.Particle),
+        v_a: u.cm / u.s,
+        n_b: u.cm**-3,
+        T_b: u.K,
+        coulomb_log: u.dimensionless_unscaled,
+        dx: float = 1e-20,
+    ):
+        self.species = species
+        self.v_a = v_a
+        self.n_b = n_b
+        self.T_b = T_b
+        self.coulomb_log = coulomb_log
+        self.dx = dx
+
+        v_0 = self._v_0()
+        x = self._x()
+        phi = self._phi(x)
+        phi_prime = self._phi_prime(x)
+
+        mu = species[0].mass / species[1].mass
```

Comment:
Confusing to call this mu, since mu is species[x].mass/ proton mass

Reply:
Hmm, do you have any suggestions for a name? It's a quantity that pops up in a few of the expressions

Reply:
maybe mass_ratio